channel/Monster3GPP38901.m

```diff
@@ -399,8 +432,15 @@ function propagateWaveforms(obj, Stations, Users, Mode)
 			end
 			
 			shadowing = obj.Channel.enableShadowing;
-			avgBuilding = mean(obj.Channel.BuildingFootprints(:,5));
-			avgStreetWidth = obj.Channel.BuildingFootprints(2,2)-obj.Channel.BuildingFootprints(1,4);
+			% Check whether we have buildings in the scenario
+			if ~isempty(obj.Channel.BuildingFootprints)
```

Comment:
This function `computePathLoss` should be merged with `computePathLossMatrix` into just `computePathLoss`. This should be a separate issue and not conflicting with the pull request.